### PR TITLE
harmonia: switch to fork

### PR DESCRIPTION
**Pull request by Mic92**
###### Description of changes

see also https://github.com/NixOS/nixpkgs/issues/221498 before merging.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/package-management/harmonia/default.nix [hunk 1]
```diff
@@ -9,16 +9,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "harmonia";
-  version = "0.2.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
-    owner = "helsinki-systems";
+    owner = "nix-community";
```
Comment (AndersonTorres) [inline pkgs/tools/package-management/harmonia/default.nix:15]: Describe this change on the commit body message

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/tools/package-management/harmonia/default.nix [hunk 2]
```diff
@@ -9,16 +9,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "harmonia";
-  version = "0.2.0";
+  version = "0.6.0";
```
[the remark this answers is not in the document]
Comment (Mic92) [reply pkgs/tools/package-management/harmonia/default.nix:12]: Done.

File: pkgs/tools/package-management/harmonia/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/tools/package-management/harmonia/default.nix:15]: Linked the issue.

**Comment by Mic92** [comment]
@AndersonTorres any more comments?

**Review by AndersonTorres — APPROVED** [review]
_Nihil Obstat_

**Comment by Mic92** [comment]
> _Nihil Obstat_

What does this mean?

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**